### PR TITLE
Update ldap tenant level to 5

### LINKED ISSUE #1378: Make ldap_tenant_level equal 5 for upgrade
**Describe the bug**
Make file in middleware/tasks/upgrade
Name accordingly
Inventory/group_vars/all and rlease_notes incremnent version number

### PR DESCRIPTION
**Pull request by alf-cactus**
closes #1378
Some cleanup is also included here
Please review the update part of this pr, I am new to this

### CODE REVIEW

File: site.yml [hunk 1]
```diff
@@ -99,14 +99,6 @@
...
 
-- hosts: all
-  roles:
-    - { role: upgrade-completion, when: "installation_mode == 'upgrade'" }
-  tags:
-    - backend
-    - frontend
-    - importer
-
 - hosts: frontends
```
Comment (tpurschke) [inline site.yml:102]: shouldn't the cleanup role run on all hosts, not just frontends?

**Review by tpurschke — CHANGES_REQUESTED** [review]
version update part looks good, for other topics see change suggestions below

File: site.yml (hunk 1 above)
Comment (alf-cactus) [reply site.yml:102]: Partially yes, it cleans up all of these parts. These tags are a combination of all the tags associated with the tasks done prior to the cleanup role

File: roles/middleware/tasks/upgrade/5.5.5.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+- name: set ldap tenant level to 5
+  postgresql_query:
+    db: "{{ fworch_db_name }}"
+    query: >
+      DO $do$ BEGIN IF EXISTS
+      (SELECT * FROM ldap_connection)
```
[the remark this answers is not in the document]
Comment (alf-cactus) [reply roles/middleware/tasks/upgrade/5.5.5.yml:6]: This was stolen from this deleted task from roles/middleware/tasks/main.yml:
[github-issue-complement.txt](https://github.com/CactuseSecurity/firewall-orchestrator/files/7756475/github-issue-complement.txt)

File: roles/api/tasks/hasura-install.yml [hunk 3]
```diff
@@ -105,8 +105,11 @@
                                         '
       HTTP_PROXY:                      "{{ http_proxy }}"
       HTTPS_PROXY:                     "{{ https_proxy }}"
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (alf-cactus) [reply roles/api/tasks/hasura-install.yml]: Done

**Comment by alf-cactus** [comment]
Works for me now: install, upgrade, remove all sample data by hand, upgrade

**Comment by alf-cactus** [comment]
Added to cleanup role: cronjobs for sample data are deleted if fortinet_demo device is not present

**Review by tpurschke — APPROVED** [review]
perfect - all tests successful

**Pull request merged**